modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/VersionedRowStore.java

```diff
@@ -48,65 +45,66 @@
  */
 public class VersionedRowStore {
```

Comment:
It seems messy that we have VersisionedRowStore and MvPartitionStorage.

Reply:
Well, no. Let it be VersionRowStore for now. Hope that we'll elaborate with better name and semantics later.